solidity/contracts/utility/TokenHolder.sol

```diff
@@ -13,31 +13,70 @@ import "./interfaces/ITokenHolder.sol";
  *
  * We consider every contract to be a 'token holder' since it's currently not possible
  * for a contract to deny receiving tokens.
- *
- * Note that we use the non standard ERC-20 interface which has no return value for transfer
- * in order to support both non standard as well as standard token contracts.
- * see https://github.com/ethereum/solidity/issues/4116
  */
 contract TokenHolder is ITokenHolder, Owned, Utils {
     using SafeERC20 for IERC20;
 
+    // prettier-ignore
+    receive() external payable override virtual {}
+
     /**
-     * @dev withdraws tokens held by the contract and sends them to an account
+     * @dev withdraws funds held by the contract and sends them to an account
      * can only be called by the owner
      *
-     * @param _token   ERC20 token contract address
-     * @param _to      account to receive the new amount
-     * @param _amount  amount to withdraw
+     * @param token ERC20 token contract address (with a special handling of NATIVE_TOKEN_ADDRESS)
+     * @param to account to receive the new amount
+     * @param amount amount to withdraw
      */
     function withdrawTokens(
-        IERC20 _token,
-        address _to,
-        uint256 _amount
-    ) public virtual override ownerOnly validAddress(address(_token)) validAddress(_to) {
-        if (_token == NATIVE_TOKEN_ADDRESS) {
-            payable(_to).transfer(_amount);
+        IERC20 token,
+        address to,
+        uint256 amount
+    ) public virtual override ownerOnly validAddress(to) {
+        safeTransfer(token, to, amount);
+    }
+
+    /**
+     * @dev withdraws multiple funds held by the contract and sends them to an account
+     * can only be called by the owner
+     *
+     * @param tokens ERC20 token contract addresses (with a special handling of NATIVE_TOKEN_ADDRESS)
+     * @param to account to receive the new amount
+     * @param amounts amounts to withdraw
+     */
+    function withdrawTokensMultiple(
+        IERC20[] calldata tokens,
+        address to,
+        uint256[] calldata amounts
+    ) public virtual override ownerOnly validAddress(to) {
+        uint256 length = tokens.length;
+        require(length == amounts.length, "ERR_INVALID_LENGTH");
```

Comment:
`ERR_INVALID_INPUT_LENGTH` IMO

Reply:
This is the error message that is used in other of our repos, so I'll keep it for now